.github/workflows/livetest.yml

```diff
@@ -0,0 +1,149 @@
+          echo "::set-output name=allowed_user::true"
+
+    outputs:
+      allowed_user: ${{ steps.check_user_perm.outputs.allowed_user }}
+
+  live-test:
+    needs: pr-info
+    if: needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/livetest')
+    runs-on: ubuntu-20.04
+    env:
+      RELAY_HOST: logs.cockpit-project.org
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - uses: actions/github-script@v3
+        id: get-pr
+        with:
+          script: |
+            const request = {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            }
+            core.info(`Getting PR #${request.pull_number} from ${request.owner}/${request.repo}`)
+            try {
+              const result = await github.pulls.get(request)
+              return result.data
+            } catch (err) {
+              core.setFailed(`Request failed with error ${err}`)
+            }
+
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ fromJSON(steps.get-pr.outputs.result).head.repo.full_name }}
+          fetch-depth: 0
+          ref: ${{ fromJSON(steps.get-pr.outputs.result).head.ref }} # or .head.ref for branch name
+
+      - name: Create a new test user
+        # WARNING: if you want to add this users to sudoers make sure to propertly delete the SINK_SSHRELAY_PRIVATE
+        run: sudo useradd --create-home livetest; echo livetest:foobar | sudo chpasswd
+
+      - name: Download dists tarball for current git SHA from GitHub
+        id: make_dist
+        continue-on-error: true
+        run: ./test/make_dist.py -d
+
+      - name: Write a comment on the PR prompting the user to try later if make_dist.py failed
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Download of the dists artifact failed: Retry again in 10 minutes'
+            })
+        if: steps.make_dist.outcome == 'failure'
+
+      - run: exit 1
+        if: steps.make_dist.outcome == 'failure'
+
+      - name: Install dependencies
+        run: |
+          sudo sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list
+          sudo apt update
+          sudo apt-get -y build-dep cockpit
+          ./tools/make-debs --quick
+          # FIXME: pcp is just utterly slow to start makes the installation fail
+          rm -f cockpit-pcp*.deb
+          for deb in cockpit*.deb; do sudo apt -y install ./$deb; done
```

Comment:
Please do `sudo apt-get install -y cockpit*.deb` rather, so that apt can resolve dependencies properly.